settings.py

```diff
@@ -266,7 +266,7 @@
     'corehq.apps.locations',
     'corehq.apps.products',
     'corehq.apps.programs',
-    'corehq.apps.registry',
+    'corehq.apps.registry.app_config.RegistryAppConfig',
```

Comment:
Why does this change?

Reply:
The app config registers the signals. For Django to find it the path here needs to be the full path the config (unless it get's put in the `apps` module but that's not how we've done it in other apps so I'm just keeping it consistent): https://docs.djangoproject.com/en/3.2/ref/applications/#configuring-applications